### PR TITLE
Update to jdk-11.0.9 and jdk-8u272

### PR DESCRIPTION
**Pull request by koenvangeerteruy**
Hello @tgeens,

I updated to the most recent versions of openjdk-8 (jdk-8u272) and openjdk-11 (jdk-11.0.9)  released on 2020-10-21.

I also updated gradle (6.7) and the gradle-plugins (5.1.0) to the last version.

In the changelog I aligned the entry numbers.

Kind regards,
Koen

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -126,7 +122,7 @@ subprojects { Project project ->
                 buildArgs.put('JAVA_VERSION', "${project.java.version.major}u${project.java.version.update}-${project.java.version.extra}");
                 buildArgs.put('JAVA_VERSION_COMPACT', "${project.java.version.major}u${project.java.version.update}${project.java.version.extra}");
             } else {
-                buildArgs.put('JAVA_VERSION', "-${project.java.version.major}.0.${project.java.version.update}+${project.java.version.extra}");
+                buildArgs.put('JAVA_VERSION', "-${project.java.version.major}.0.${project.java.version.update}+${project.java.version.extra}.1");
```
Comment (vierbergenlars) [inline build.gradle:125]: Shouldn't the `.1` here be a part of `java.version.extra`?

**Comment by tgeens** [comment]
Thanks @koenvangeerteruy

**Review by tgeens — APPROVED** [review]

**Review by vierbergenlars — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (tgeens) [reply build.gradle:125]: but at that point `JAVA_VERSION_COMPACT` below no longer matches ?

Comment (tgeens) [reply build.gradle:125]: needs more restructuring to actually fix that

Comment (koenvangeerteruy) [reply build.gradle:125]: I don't know whether it is worth the effort to restructure. I think it was related to a rebuild, so the artifact ended up in a different download folder.
The tar.gz-file is still 11.0.9+11 and the Java version as well.
I would wait to make changes and see how the next release is

Comment (anghelutar) [reply build.gradle:125]: Shouldn't these changes propagate to docker-tomcat?

Bringing java version up-to-date in:
https://github.com/xenit-eu/docker-tomcat/blob/master/tomcat-7.0/jdk-7-centos-7.gradle
https://github.com/xenit-eu/docker-tomcat/blob/master/tomcat-7.0/jdk-7-trusty.gradle
https://github.com/xenit-eu/docker-tomcat/blob/master/tomcat-7.0/jdk-8-bionic.gradle
https://github.com/xenit-eu/docker-tomcat/blob/master/tomcat-8.5/jdk-11-ubuntu-18.04.gradle

Comment (koenvangeerteruy) [reply build.gradle:125]: @anghelutar Yes, I think that would be a good idea. Will create a PR in docker-tomcat.

**Pull request merged**